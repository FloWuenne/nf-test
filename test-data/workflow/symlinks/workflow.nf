workflow TEST {
    take:
        myFile

    main:
        myFile.view()

    emit:
        myFile
}