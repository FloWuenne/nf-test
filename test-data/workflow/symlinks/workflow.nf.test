nextflow_workflow {

    name "Test Workflow TEST"
    script "./workflow.nf"
    workflow "TEST"

    test("Should run without failures") {

        when {
            workflow {
                """
                // define inputs of the workflow here. Example:
                input[0] = Channel.of(
                    file("${moduleDir}/test-file2.txt", checkIfExists: true)
                )
                """
            }
        }

        then {
            assert workflow.success
            assert snapshot(workflow.out).match()
        }

    }

}