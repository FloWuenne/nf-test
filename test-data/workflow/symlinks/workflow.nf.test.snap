{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "test-file.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                ],
                "myFile": [
                    "test-file.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.0-rc2",
            "nextflow": "23.04.3"
        },
        "timestamp": "2024-07-13T12:52:26.454023"
    }
}